src/plugins/FileService/Preview.tsx

```diff
@@ -0,0 +1,111 @@
+import { makeStyles, Typography, Theme, Paper } from '@material-ui/core'
+import React from 'react'
+import { CopyableCode } from './components/Copyable'
+import type { FileInfo } from './types'
+import { formatFileSize } from './utils'
+import { Image } from '../../components/shared/Image'
+import { getUrl } from '../../utils/utils'
+
+interface Props {
+    info: FileInfo
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+    root: {
+        display: 'flex',
+        alignItems: 'center',
+        width: 345,
+        borderWidth: 1,
+        borderStyle: 'solid',
+        borderColor: theme.palette.divider,
+        boxSizing: 'border-box',
+        borderRadius: 12,
+        padding: 16,
+        cursor: 'default',
+        userSelect: 'none',
+        '& p': { margin: 0 },
+    },
+    meta: {
+        flex: 1,
+        minWidth: '1%',
+        marginLeft: 18,
+        marginRight: 18,
+    },
+    name: {
+        fontSize: 16,
+        lineHeight: 1.85,
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+    },
+    size: {
+        fontSize: 14,
+        lineHeight: 1.85,
+    },
+    unencrypted: {
+        fontSize: 14,
+        lineHeight: 1.85,
+    },
+    encrypted: {
+        fontSize: 14,
+        lineHeight: 1.85,
+    },
+    code: {
+        lineHeight: 1,
+        userSelect: 'auto',
+    },
+    download: {
+        display: 'block',
+        cursor: 'pointer',
+    },
+}))
+
+export const Preview: React.FC<Props> = ({ info }) => {
+    const classes = useStyles()
+    const fileKey = info.key ? (
+        <Typography component="p" color="textPrimary" className={classes.encrypted}>
+            File Key: <CopyableCode className={classes.code}>{info.key}</CopyableCode>
+        </Typography>
+    ) : (
+        <Typography component="p" color="textSecondary" className={classes.unencrypted}>
+            This file is not encrypted
+        </Typography>
+    )
+    const link = `https://arweave.net/${info.landingTxID}`
+    const onClick = (event: React.MouseEvent) => {
+        event.preventDefault()
+        event.stopPropagation()
+        if (info.key) {
+            open(`${link}#${info.key}`)
+        } else {
+            open(link)
+        }
+    }
+    return (
+        <Paper elevation={0} className={classes.root}>
+            <Image
+                className={classes.download}
+                src={getUrl('/plugin/file-service/preview-file.svg')}
```

Comment:
Please don't use this utility to render SVG. You can change the SVG into a TSX file and get a high-resolution icon. And if possible, please use the icon from [@material-ui/icons](https://material-ui.com/components/material-icons/) or [react-feather](https://feathericons.com/).